src/sentry/static/sentry/app/components/pluginConfig.jsx

```diff
@@ -81,6 +82,27 @@ const PluginConfig = React.createClass({
     });
   },
 
+  testPlugin() {
+    let loadingIndicator = IndicatorStore.add(t('Sending test..'));
+    this.api.request(this.getPluginEndpoint(), {
+      method: 'POST',
+      data: {
+        test: true
+      },
+      success: data => {
+        this.setState({testResults: JSON.stringify(data)});
```

Comment:
ok last little thing, instead of stringifying all of data, how about doing `JSON.stringify(data.detail)` since i think that's the closest to what the non-react version was? and then it's also just whatever the plugin returns.